src/format/strftime.rs

```diff
@@ -668,18 +675,26 @@ fn test_strftime_docs() {
     assert_eq!(dt.format("%%").to_string(), "%");
 }
 
+/// helper function to setup a date time for the localized tests
+#[cfg(feature = "unstable-locales")]
+#[cfg(test)]
+fn setup_naive_dt() -> crate::DateTime<crate::FixedOffset> {
```

Comment:
Wait, this is returning a `DateTime`, not a `NaiveDateTime`.
Should this be renamed `setup_datetime()` or perhaps `create_datetime_for_localized_tests()`?